front/app/containers/Admin/projects/project/general/components/ProjectNameInput.tsx

```diff
@@ -27,6 +28,27 @@ const ProjectNameInput = ({
   apiErrors,
   handleTitleMultilocOnChange,
 }: Props) => {
+  const { formatMessage } = useIntl();
+  const dedupedErrors = dedupApiErrors(apiErrors.title_multiloc);
```

Comment:
Seems like potentially the component should handle this, otherwise we need to add this code every time we use it?